internal/blobstore/blobstore_test.go

```diff
@@ -279,7 +279,7 @@ func (s *blobStoreSuite) TestPutPartAgain(c *gc.C) {
 	// Perform a Put with mismatching content. This should leave the part in progress
 	// but not completed.
 	err := s.store.PutPart(id, 0, strings.NewReader("something different"), int64(len(content)), 0, hashOf(content))
-	c.Assert(err, gc.ErrorMatches, `cannot upload part ".+": hash mismatch`)
```

Comment:
I guess the new version of goose does not allow us to be a bit more specific when checking the error?